distribution/lib/Standard/Test/0.0.0-dev/src/Bench.enso

```diff
@@ -122,7 +122,7 @@ type Bench
                 validate_name name
                 conf.validate
                 ensure_distinct_names <| specs.map (_.name)
-            Bench.Spec name code ->
+            Bench.Spec name code _ _ ->
```

Comment:
This change is only necessary because _adding additional fields_ to non-private constructors is an **incompatible change**.